src/cr/cube/matrix/measure.py

```diff
@@ -2732,7 +2770,8 @@ def _weighted_median(sorted_counts, sorted_values):
         # --- TODO: Does this behave okay for large numbers (or is precision a problem)?
         if cumulative_prop[median_idx] == 0.5:
             return np.mean(sorted_values[[median_idx, median_idx + 1]])
-        return sorted_values[median_idx]
+        else:
```

Comment:
did you change all this stuff for some reason? Sorry just out of curiosity maybe we decided something and I forgot